publicly-available-specification/document.adoc

```diff
@@ -0,0 +1,176 @@
+:circulated-date:
+:comment: ### Directory in which images are located: all local image file locations are prefixed with this directory. (Optional.)
+:imagesdir: images
+:comment: ### Name of this AsciiDoc file
+:docfile: document.adoc
+:comment: ### Metanorma flavour: applies to all ISO documents
+:mn-document-class: iso
+:comment: ### Desired output formats. Metanorma-ISO supports the following output extension values: xml (MN XML), html (ISO-style HTML), html_alt (pretty HTML), doc (Word .doc)
+:mn-output-extensions: xml,html,doc,html_alt
+:local-cache-only:
+:data-uri-image:
+
+
+////
+Comment 1: DOCUMENT NUMBER (:docnumber:)
+
+Sequence of digits that gives the document number value included in the document identifier, for example: 
+
+99999 (to author ISO-99999)
+8000 (to author ISO-8000), 
+22745 (to author ISO-22745), 
+29002 (to author ISO-29002)
+////
+
+////
+Comment 2: DOCUMENT TITLE (mandatory)
+
+The document title in Metanorma may be composed by three attributes from which two are optional and one mandatory. These attributtes are used instead of the metanorma-standoc `:title:` attribute and the default AsciiDoc title (the first line of the document header, prefixed with `=`):
+
+`:title-intro-en:` The introductory component of the English title (optional)
+`:title-main-en:` The main component of the English title (mandatory)
+`:title-part-en:` The English title of the document part (optional)
+`:title-intro-fr:` The introductory component of the French title (optional)
+`:title-main-fr:` The main component of the French title (mandatory)
+`:title-part-fr:` The French title of the document part (optional)
+
+Both the English and the French titles for the document are required, for both the English and the French version of the document. The main title will be that of the language the document is in.
+////
+
+////
+Comment 3: DOCUMENT TYPE (:doctype:) (mandatory)
+
+Its possible values are defined by ISO deliverables: https://www.iso.org/deliverables-all.html 
+
+The permitted types are: 
+international-standard, 
+technical-specification, 
+technical-report, 
+publicly-available-specification, 
+international-workshop-agreement, 
+guide.
+////
+
+////
+Comment 4:  DOCUMENT STAGE (:docstage:) (Synonym of :status:)
+
+Stage code for the document status. See International harmonized stage codes: https://www.iso.org/stage-codes.html
+
+ISO authors usually create Metanorma documents at stages 00 through 40 (PWI, NWIP, WD, CD, DIS). Documents at the final stages, 50 and 60 (FDIS, PRF, IS), are created by ISO Editorial Program Managers at ISO/CS (aka ISO EPMs, ISO Editors). Authors using Metanorma are not expected to edit documents at those stages, and are not necessary in a normal submission process. These stages, however, can be used for mirroring and tracking of final stage and published standards, which many authors do.
+
+Example values:
+PWI: 00
```

Comment:
Please see this diagram for possible stages for different deliverable types. We need to also put this on metanorma.com:

![](https://www.iso.org/sites/directives/current/consolidated/image/Dir-1_ed11figF_e.jpg)

Reply:
I have added additional information about possible stages and substages according to the deliverable type.